### DISCUSSION TITLE
Office.com 圖示重影

### DISCUSSION THREAD

**Discussion by cwl-uml** [F9y4ng/GreasyFork-Scripts#192] [问答专区(Question / Answer)] [answered]
如圖所示，打開渲染後office圖示會有重影，關掉腳本後則正常

![icon](https://user-images.githubusercontent.com/38968787/230726786-5e39d984-b7a1-4683-9961-eb9ea8967bbc.png)

**Comment by F9y4ng** [comment]
不论是你的截图还是原页面，我并没有看到有重影的文字呢？ 

你把有重影的部分放大截图，最好用红框圈一下。

**Reply by cwl-uml** [reply]
重影的不是文字，是圖示

沒有開啟腳本
![icon-2](https://user-images.githubusercontent.com/38968787/230732703-ae241fc2-c4ef-46ab-8767-16c531e3f3db.png)

開啟腳本
![icon](https://user-images.githubusercontent.com/38968787/230732720-b579a0b8-9e31-4fba-bf34-7047cfa7f539.png)

是不影響閱讀，不過看著還是覺得怪

**Comment by F9y4ng** [comment]
在**排除渲染的HTML标签**中追加 ~`,.o365cs-base [class^=ms-Icon--]`~，保存为站点独享数据即可。

**Reply by cwl-uml** [reply]
我這邊就算添加了排除，圖示還是重影狀態

![image](https://user-images.githubusercontent.com/38968787/230763320-d84f3841-1b39-456d-9012-153c052a8484.png)

**Reply by F9y4ng** [reply]
我想应该是简体中文和繁体中文的应用的样式不同造成吧？

你可以根据我给你的截图，F12打开控制台，按ctrl+shift+C打开元素选择器，选中图标后，将元素代码部分截图给我看看。
![test](https://user-images.githubusercontent.com/15855068/230766854-1818adab-eb36-42f9-9188-8575727b48f1.png)

**Reply by cwl-uml** [reply]
應用程式圖示
![icon](https://user-images.githubusercontent.com/38968787/230772012-e708937b-37c1-4fe2-bde0-0faf24dace96.png)

側邊欄App圖示
![sideIcon](https://user-images.githubusercontent.com/38968787/230772016-c9780a9b-1b77-487c-aa06-3e505319ab36.png)

**Comment by F9y4ng** [comment, marked as answer]
确实样式有差异，在排除渲染的HTML标签中追加 `,.o365cs-base [class*='ms-Icon--']`，保存为站点独享数据即可。

**Reply by cwl-uml** [reply]
已解決，感謝你~

**Reply by F9y4ng** [reply]
该样式问题已收录至 [网站样式错误修正的设置分享，不定期更新，自取自用 #42 ](https://github.com/F9y4ng/GreasyFork-Scripts/discussions/42)
